types/src/model/blockchain.rs

```diff
@@ -621,3 +621,57 @@ pub struct ReceiveActivity {
     /// The ScriptPubKey, converted to `model::ScriptPubkey`.
     pub output_spk: ScriptPubkey,
 }
+
+/// Models the result of the JSON-RPC method `scantxoutset`.
+#[derive(Deserialize, Debug, Clone, PartialEq)]
+#[serde(untagged)]
+pub enum ScanTxOutSet {
+    Start(ScanTxOutSetStart),
+    Abort(bool),
+    Status(Option<ScanTxOutSetStatus>),
+}
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct ScanTxOutSetStart {
+    /// Whether the scan is completed. For v19 onwards.
+    pub success: Option<bool>,
+    /// The number of unspent transaction outputs scanned. For v19 onwards.
+    pub txouts: Option<u64>,
+    /// The current block height (index). For v19 onwards.
+    pub height: Option<u64>,
+    /// The hash of the block at the tip of the chain. For v19 onwards.
+    pub bestblock: Option<BlockHash>,
+    /// The unspents
+    pub unspents: Vec<ScanTxOutSetUnspent>,
+    /// The total amount of all found unspent outputs in BTC
+    pub total_amount: Amount,
+}
+
+#[derive(Deserialize, Debug, Clone, PartialEq)]
+pub struct ScanTxOutSetStatus {
```

Comment:
Observation: This struct never  changes

Use `git grep -A 4 'pub struct ScanTxOutSetStatus'` to see.